static/main.js

```diff
@@ -450,6 +450,14 @@ function start() {
         }
     }
 
+    _.each(options.languages, function (lang) {
+        try {
+            lang.logoData = require('../views/resources/logos/' + lang.logoUrl);
```

Comment:
There has to be a better way, right?

Reply:
Actually, ADA's gets converted into an url of the form `"/static/292f9fd172ecc1d55151.png"` so it might just work liks this?

Reply:
Is there a need for a try-catch here? The assets should exist so I don't know if require() would ever fail here

Reply:
I put it in there just so in case they don't, it shows a broken image icon instead of crashing the whole page

Reply:
I bet there's some magical webpack-fu that can combine all the logos in some way.... but if this works I'm happy for now. That existing webpack stuff unconditionally bundles all images into content-hash-named files (Which is cool for caching etc).